app/controllers/instruments_controller.rb

```diff
@@ -19,47 +19,57 @@ def index
     @instruments.each { |i| @reservations[i.id] = i.reservations.upcoming[0..2]}
   end
 
-  # GET /instruments/1
+  # GET /facilities/:facility_id/instruments/:instrument_id
   def show
```

Comment:
This method makes me cry. I'm not sure if it's worth taking the time now, but it would be nice to extract a lot of this logic into its own class.